packages/web/components/complex/all-pools-table-set.tsx

```diff
@@ -245,7 +251,7 @@ export const AllPoolsTableSet: FunctionComponent<{
       sortedAllPoolsWithMetrics,
     ] = useSortedData(tvlFilteredPools, initialKeyPath, initialSortDirection);
 
-    const [query, setQuery, filteredPools] = useFilteredData(
+    const [query, do_setQuery, filteredPools] = useFilteredData(
```

Comment:
Is there a reason why we add the `do_` prefix?

Reply:
We have a wrapper function that performs additional operations before that core function is called. It's somewhat of a convention in our app, we use it a lot to log amplitude events along with the core function.

Reply:
I feel the `do` makes it seem like snake case instead of camel. Maybe we could remove it it and leave the `_` like a private method. What do you think?

Reply:
Sure, think we should do it elsewhere in the app?